config/application.rb

```diff
@@ -76,6 +76,9 @@ class Application < Rails::Application
     config.external_url_mapping = APP_CONFIG = YAML.load(File.read(Rails.root + "config/external_link_mapping.yml"))
 
     config.before_initialize do
+      Settings.add_source!(Rails.root.join('config', 'hets.yml').to_s)
+      Settings.reload!
+
```

Comment:
Why reload? If you always wanna include hets.yml (for whatever reason), it should be made clear  by adding a corresponding 'include' directive in settings.yml.